application/src/main/scala/com/azavea/franklin/api/Server.scala

```diff
@@ -69,66 +74,71 @@ $$$$
-        IO
-      ](
-        apiConfig.enableTiles
-      ).endpoints
-      docs      = allEndpoints.toOpenAPI("Franklin", "0.0.1")
-      docRoutes = new SwaggerHttp4s(docs.toYaml, "open-api", "spec.yaml").routes[IO]
-      searchRoutes = new SearchService[IO](
-        apiConfig.apiHost,
-        apiConfig.defaultLimit,
-        apiConfig.enableTiles,
-        xa
-      ).routes
-      tileRoutes = new TileService[IO](apiConfig.apiHost, apiConfig.enableTiles, xa).routes
-      collectionRoutes = new CollectionsService[IO](xa, apiConfig).routes <+> new CollectionItemsService[
-        IO
-      ](
-        xa,
-        apiConfig
-      ).routes
-      router = CORS(
-        Router(
-          "/" -> ResponseLogger.httpRoutes(false, false)(
-            collectionRoutes <+> searchRoutes <+> tileRoutes <+> docRoutes
-          )
+    AsyncHttpClientCatsBackend.resource[IO]() flatMap { implicit backend =>
+      for {
+        connectionEc  <- ExecutionContexts.fixedThreadPool[IO](2)
+        transactionEc <- ExecutionContexts.cachedThreadPool[IO]
+        xa <- HikariTransactor.newHikariTransactor[IO](
+          "org.postgresql.Driver",
+          dbConfig.jdbcUrl,
+          dbConfig.dbUser,
+          dbConfig.dbPass,
+          connectionEc,
+          Blocker.liftExecutionContext(transactionEc)
         )
-      ).orNotFound
-      serverBuilderBlocker <- Blocker[IO]
-      server <- {
-        BlazeServerBuilder[IO](serverBuilderBlocker.blockingContext)
-          .bindHttp(apiConfig.internalPort.value, "0.0.0.0")
-          .withConnectorPoolSize(128)
-          .withBanner(banner)
-          .withHttpApp(router)
-          .resource
+        collectionItemEndpoints = new CollectionItemEndpoints[IO](
+          apiConfig.defaultLimit,
+          apiConfig.enableTransactions,
+          apiConfig.enableTiles
+        )
+        collectionEndpoints = new CollectionEndpoints[IO](
+          apiConfig.enableTransactions,
+          apiConfig.enableTiles
+        )
+        allEndpoints = collectionEndpoints.endpoints ++ collectionItemEndpoints.endpoints ++ new SearchEndpoints[
+          IO
+        ].endpoints ++ new TileEndpoints[
+          IO
+        ](
+          apiConfig.enableTiles
+        ).endpoints
+        docs      = allEndpoints.toOpenAPI("Franklin", "0.0.1")
+        docRoutes = new SwaggerHttp4s(docs.toYaml, "open-api", "spec.yaml").routes[IO]
+        searchRoutes = new SearchService[IO](
+          apiConfig.apiHost,
+          apiConfig.defaultLimit,
+          apiConfig.enableTiles,
+          xa
+        ).routes
+        tileRoutes = new TileService[IO](apiConfig.apiHost, apiConfig.enableTiles, xa).routes
+        itemExtensions       <- Resource.liftF { itemExtensionsRef[IO] }
+        collectionExtensions <- Resource.liftF { collectionExtensionsRef[IO] }
+        collectionRoutes = new CollectionsService[IO](xa, apiConfig, collectionExtensions).routes <+> new CollectionItemsService[
```

Comment:
Right below the line `115`, or somewhere above we can make an instnace of a logger:

```scala
collectionExtensions <- Resource.liftF { collectionExtensionsRef[IO] }
// implicit0 is a syntax provided by the bm4 plugin that is used in this project
implicit0(logger: Logger[IO]) = Slf4jLogger.getLogger[IO]
```

Reply:
2e5928f